pmoves/services/agentgym-rl-coordinator/coordinator/publisher.py

```
@@ -0,0 +1,373 @@
+
+        Returns:
+            Publication result with dataset_id and repo_url
+
+        Raises:
+            ValueError: If HF_TOKEN not configured or huggingface_hub not installed
+            RuntimeError: If publishing fails
+        """
+        if not self.hf_token:
+            raise ValueError(
+                "HF_TOKEN not configured. Set HF_TOKEN environment variable."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)